src/templates/profile-block/index.vue

```diff
@@ -331,7 +331,10 @@
     {
       label: 'Teams Permissions',
       to: '/teams-permission'
-    }
+    },
+    ...(hasAccessToSSOManagement.value
```

Comment:
Essa parte aqui, não está legal não ta dificil de entender, não seria melhore criar um method, exemplo:  
`
checkHasAccessToSSOManagement = () => {
 if(hasAccessToSSOManagement.value) {
      profile.value.push({
         label: 'SSO Management', to: '/identity-providers'
      })
   }
}
` e esse method ser chamado em uma onMounted, assim ficaria bem melhor de entender

Reply:
não vejo pq criar mais um metodo e ter que usar onMount só para isso